.github/CODEOWNERS

```diff
@@ -136,12 +159,52 @@ src/applications/mhv/secure-messaging @department-of-veterans-affairs/vfs-mhv-se
 src/applications/mhv/landing-page @department-of-veterans-affairs/vfs-mhv-integration @department-of-veterans-affairs/va-cto-health-products
 src/applications/avs @department-of-veterans-affairs/va-cto-health-products
 
-# VA.gov Product Teams - Forms
+# Dependents Benefits
+src/applications/static-pages/dependency-verification @department-of-veterans-affairs/benefits-ce-dependents
+src/applications/static-pages/view-modify-dependent @department-of-veterans-affairs/benefits-ce-dependents
+src/applications/static-pages/view-modify-dependents @department-of-veterans-affairs/benefits-ce-dependents
 
-src/applications/simple-forms @department-of-veterans-affairs/platform-va-product-forms
+# Flagship Mobile Reviewers
+src/applications/static-pages/subscription-creators @department-of-veterans-affairs/flagship-mobile-reviewers
```

Comment:
Why isn't this recognized as a VA Github team?

Reply:
I suspect there's maybe a repo-level setting, to add this team to vets-website. (Mobile has their own repos.)

Reply:
Looking into this with Platform